src/main/java/software/amazon/qldb/QldbDriverBuilder.java

```diff
@@ -26,14 +26,14 @@
  * <pre>{@code
  *     QldbSessionClientBuilder sessionClientBuilder = QldbSessionClient.builder();
  *
- *     QldbDriverBuilder builder = QldbDriver
- *          .builder()
- *          .ledger(ledger);
- *          .maxConcurrentTransactions(poolLimit)
- *          .transactionRetryPolicy(TransactionRetryPolicy.builder().maxNumberOfRetries(retryLimit).build());
- *          .sessionClientBuilder(sessionClientBuilder)
- *          .httpClientBuilder(ApacheHttpClient.builder().maxConnections(maxConnections))
- *          .build();
+ *     QldbDriver qldbDriver = QldbDriver
+ *         .builder()
```

Comment:
Updated indentation to 4 spaces to follow Oracle's [code conventions](https://www.oracle.com/java/technologies/javase/codeconventions-indentation.html)